Formula/node.rb

```diff
@@ -47,6 +47,12 @@ def install
     # These symlinks are never used & they've caused issues in the past.
     rm_rf libexec/"share"
 
+    # suppress incorrect node incompatibility warning from npm
+    # remove during next npm upgrade
+    inreplace libexec/"lib/node_modules/npm/lib/utils/unsupported.js",
+      "{ver: '12', min: '12.0.0'}",
+      "{ver: '12', min: '12.0.0'}, {ver: '13', min: '13.0.0'}"
```

Comment:
Is this still needed with the latest npm release?

Reply:
It's not needed with the latest `npm` v6.12.1 released 3 hours ago, but this isn't in node upstream yet.